app/assets/stylesheets/application.scss

```
@@ -1,43 +1,41 @@
-/*
- *= require_tree .
- *= require_self
- */
+
```

ここはコメントではありません。他のcssファイルが読み込めなくなりますので，残して下さい。
```suggestion
/*
 *= require_tree .
 *= require_self
 */
```